FLEDGE_extended_PA_reporting.md

```diff
@@ -160,14 +160,13 @@ function generateBid(...) {
       value: 1
```

Comment:
Can we change the 255 to 500 or some rounder number that makes more sense as a base bucket offset?